README.md

```diff
@@ -264,6 +265,35 @@ Returns a single post, specified by its Walls.io post id.
 }
 ```
 
+### PUT api/posts/*{postId}*.*{format}*
+
+Changes a single post's status or highlighting, specified by its Walls.io post id. Use this endpoint if you want to highlight or hide/unhide a post on the Wall.
+
+#### Example request
+`PUT https://walls.io/api/posts/17824236.json?access_token=<YOUR_ACCESS_TOKEN>`
+
+#### Parameters
+- `access_token` *(required)*: Your Walls.io access token.
+- `is_highlighted`: Indicates if the post should be highlighted on the Wall.
```

Comment:
-> "Set this to `1` if the post should be highlighted, or `0` to remove an existing highlight flag."